manifests/init.pp

```diff
@@ -50,6 +50,8 @@
 
   $proxy_server           = undef,
   $proxy_type             = undef,
+  $proxy_host             = undef,
+  $proxy_port             = undef,
 
```

Comment:
I updated to use the same proxy_uri building as the core params uses. The Shibboleth config requires host and port be set separately. I consider it less error prone to assemble the proxy uri than to try to split a URL into host and port.